legacy/src/app/partials/nodes-list.html

```diff
@@ -716,9 +716,13 @@ <h4 class="page-header__dropdown-title">Add rack controller</h4>
             <pre><code># To add a new rack controller, SSH into the rack controller.
 # Install the maas-rack-controller package.
 sudo apt install maas-rack-controller
+# Or if you use snap
+sudo snap install maas
 # Register the rack controller with this MAAS. If the rack controller (and machines)
 # don't have access to the URL, use a different IP address to allow connection.
-sudo maas-rack register --url {$ tabs.controllers.registerUrl $} --secret {$ tabs.controllers.registerSecret $}</code></pre>
+sudo maas-rack register --url {$ tabs.controllers.registerUrl $} --secret {$ tabs.controllers.registerSecret $}
+# Or if you use snap
+sudo /snap/maas/current/bin/maas-rack register --url {$ tabs.controllers.registerUrl $} --secret {$ tabs.controllers.registerSecret $}</code></pre>
```

Comment:
Why this maas-rack command is not in the system path like the mass command? The snap should be fixed to add it to the path.

Reply:
This is not the expected way to register a rackcontroller in the snap. The `maas` CLI should be used as:

```
sudo maas init rack --maas-url $url --secret $secret
```

For reference, this is the CLI help:

```
$ maas init rack --help
usage: maas init rack [-h] [--maas-url MAAS_URL] [--secret SECRET] [--force]

Initialise MAAS to run only a rack controller.

optional arguments:
  -h, --help           show this help message and exit
  --maas-url MAAS_URL  URL that MAAS should use for communicate from the nodes
                       to MAAS and other controllers of MAAS.
  --secret SECRET      Secret token required for the rack controller to talk
                       to the region controller(s). Only used when in 'rack'
                       mode.
  --force              Skip confirmation questions when initialization has
                       already been performed.
```